NEWS

```diff
@@ -1,3 +1,66 @@
+KLEE 2.2, 7 December 2020
+=========================
+
+Incorporating changes from 4 March 2020 to 7 December 2020.
+Maintainers during this time span: @ccadar and @MartinNowack
+Documentation at http://klee.github.io/releases/docs/v2.2
+
+== Major features ==
+- Added support for C++ exceptions (@corrodedHash, @jbuening, @futile)
```

Comment:
If you could, please move me to the back of the list, it is safe to say that @futile put much more time and effort into this than I did.